frontend/ui/src/components/projects/roadmap-view/components/project-detail.tsx

```
@@ -0,0 +1,180 @@
+                  {project.subProjects?.length} sub-projects with an average
+                  progress of {subProjectsProgress}%
+                </CardDescription>
+              </CardHeader>
+              <CardContent>
+                <div className="space-y-6">
+                  {project.subProjects?.map((subProject) => (
+                    <div
+                      key={subProject.id}
+                      className="cursor-pointer rounded-md border p-4 transition-colors hover:bg-gray-50"
+                      onClick={() => onSelectSubProject(subProject)}
+                    >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)